LEGO1/lego/legoomni/include/legocarbuild.h

```diff
@@ -73,6 +73,17 @@ typedef LegoVehicleBuildState LegoJetskiBuildState;
 // SIZE 0x34c
 class LegoCarBuild : public LegoWorld {
 public:
+	// SIZE 0x1c
+	struct LookupTableData {
+		undefined2 m_unk0x00; // 0x00
```

Comment:
The different sizes of the members are really odd considering they are all just script/entity IDs, but I guess it's verified to be correct?

Reply:
Your comment made me investigate a bit more and I managed to get a 100 % match. While changing the data types did improve the match a while ago, it didn't end up mattering for the final result. For example, the code is identical for `undefined2 m_unk0x00` and `undefined4 m_unk0x00`.